src/index.js

```diff
@@ -778,4 +778,81 @@ export default class NestedList {
      */
     this.unshiftItem();
   }
+
+  /**
+   * On paste callback that is fired from Editor
+   *
+   * @param {PasteEvent} event - event with pasted data
+   */
+  onPaste(event) {
+    const list = event.detail.data;
+
+    this.data = this.pasteHandler(list);
```

Comment:
if the method returns something, name it like `getSomethingFrom...()`

Reply:
Renamed getDataFromPaste